diesel_migrations/migrations_macros/src/embed_migrations.rs

```diff
@@ -78,7 +78,13 @@ pub fn derive_embed_migrations(input: &syn::DeriveInput) -> quote::Tokens {
 }
 
 fn migration_literals_from_path(path: &Path) -> Result<Vec<quote::Tokens>, Box<Error>> {
-    try!(migration_paths_in_directory(path))
+    let mut migrations = migration_paths_in_directory(path)?;
+
+    // Sort the migrations to make sure embedded migrations are deterministic
```

Comment:
These sort of comments tend to get out of date with the code, and we prefer not to have them. Context like this is better kept in the commit message instead.

Reply:
I've moved the comment to the commit message.